### PR TITLE
School and Provider user journey for deleting another user made more consistent (remove -> delete)

### PR DESCRIPTION
**Pull request by dp-daly**
## Context

- Same design context as [#927.](https://github.com/DFE-Digital/itt-mentor-services/pull/927)
- This PR encompasses two trello tickets as they share some tests.

### School user

**User show page:**

- Update “Remove user” to “Delete user”

**Remove user confirmation page:**

- Update “Are you sure you want to remove this user?” to “Are you sure you want to delete this user?”

- Add paragraph text above the warning component:“%{user_name} will no longer be able to view, create or manage placements at your school.”

- Update the warning component text from “removed” to “deleted”.

- Update the button text from “Remove user” to “Delete user”

### Provider user

**User show page:**

- Update “Remove user” to “Delete user”

**"Remove user" confirmation page:**

- Update “Are you sure you want to remove this user?” to “Are you sure you want to delete this user?”
- Add paragraph text above the warning component:“%{user_name} will no longer be able to access placement information.”
- Update the warning component text “removed” to “deleted”
- Update button text from “Remove user” to “Delete user”

## Guidance to review

- Review school user journey to delete another user, i.e. Mary.
- Review provider user journey to delete another user.
- Run `bundle exec rspec spec/system/placements`.

## Link to Trello card

https://trello.com/c/gTlRJQx2/666-improve-school-user-user-journey-content
https://trello.com/c/stZYPyzG/669-improve-provider-user-user-content

### CODE REVIEW

File: config/locales/en/placements/providers/users.yml [hunk 1]
```diff
@@ -8,14 +8,15 @@ en:
...
-          are_you_sure: Are you sure you want to remove this user?
-          remove_user: Remove user
+          page_title: Are you sure you want to delete this user? - %{user_name}
+          are_you_sure: Are you sure you want to delete this user?
+          delete_user: Delete user
           cancel: Cancel
           change_organisation: Change organisation
-          user_will_be_sent_an_email: The user will be sent an email to tell them you removed them from %{organisation_name}
+          user_will_no_longer: '%{user_name} will no longer be able to access placement information.'
+          user_will_be_sent_an_email: The user will be sent an email to tell them you deleted them from %{organisation_name}
```
Comment (JamieCleare2525) [inline config/locales/en/placements/providers/users.yml:21]:
```suggestion
          user_will_be_sent_an_email: The user will be sent an email to tell them you deleted them from %{organisation_name}.
```
Missing full stop.

File: config/locales/en/placements/schools/users.yml [hunk 2]
```diff
@@ -8,14 +8,15 @@ en:
...
-          are_you_sure: Are you sure you want to remove this user?
-          remove_user: Remove user
+          page_title: Are you sure you want to delete this user? - %{user_name}
+          are_you_sure: Are you sure you want to delete this user?
+          delete_user: Delete user
           cancel: Cancel
           change_organisation: Change organisation
-          user_will_be_sent_an_email: The user will be sent an email to tell them you removed them from %{organisation_name}
+          user_will_no_longer: "%{user_name} will no longer be able to view, create or manage placements at your school."
+          user_will_be_sent_an_email: The user will be sent an email to tell them you deleted them from %{organisation_name}
```
Comment (JamieCleare2525) [inline config/locales/en/placements/schools/users.yml:21]:
```suggestion
          user_will_be_sent_an_email: The user will be sent an email to tell them you deleted them from %{organisation_name}.
```
Missing full stop.

File: config/locales/en/placements/schools/users.yml [hunk 3]
```diff
@@ -8,14 +8,15 @@ en:
...
         you_cannot_perform_this_action: You cannot perform this action
```
Comment (JamieCleare2525) [inline config/locales/en/placements/schools/users.yml:22]: I think this can be removed.

File: config/locales/en/placements/providers/users.yml [hunk 4]
```diff
@@ -8,14 +8,15 @@ en:
...
         you_cannot_perform_this_action: You cannot perform this action
```
Comment (JamieCleare2525) [inline config/locales/en/placements/providers/users.yml:22]: I think this can be removed.

Comment (dp-daly) [reply config/locales/en/placements/providers/users.yml:22]: Hmm the design says to update the warning text from removed -> deleted.

File: config/locales/en/placements/schools/users.yml (hunk 3 above)
Comment (dp-daly) [reply config/locales/en/placements/schools/users.yml:22]: Hmm the design says to update the warning text from removed -> deleted.

**Review by JamieCleare2525 — APPROVED** [review]

**Review by Kizr — APPROVED** [review]

**Pull request merged**